### PR TITLE
Update libxml installation text

### LINKED ISSUE #1362: The LibXML installation page is not up to date
From manual page: https://php.net/libxml.installation

The following configure option doesn't exist anymore : --with-libxml-dir
The following seems fine : --with-libxml

Thanks
---

### LINKED ISSUE #2574: Documentation of unsupported configure flag `--with-libxml-dir`
The page https://php.net/libxml.installation states that the directory to libxml2 can be configured using `--with-libxml-dir=[DIR]`, but this option has been removed as of PHP 7.4 RC1 (see [29d1b7fd](https://github.com/php/php-src/commit/29d1b7fd521af288e9f04d784f59a2d15b494a30))

### PR DESCRIPTION
**Pull request by nielsdos**
Fixes #1362
Fixes #2574

### CODE REVIEW

File: reference/libxml/setup.xml [hunk 1]
```diff
@@ -19,38 +19,15 @@
...
-    <para>
-     <filename>/usr/local/bin/xml2-config</filename>
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     <filename>/usr/bin/xml2-config</filename>
-    </para>
-   </listitem>
-  </orderedlist>
```
Comment (Girgias) [inline reference/libxml/setup.xml:49]: Actually... since when does libxml use pkg-config? As the manual must document PHP 7.0 onwards

Comment (nielsdos) [reply reference/libxml/setup.xml:49]: As of 7.4 it uses pkg-config. I didn't know we documented EOL stuff too.
So this is going to be inconvenient

Comment (Girgias) [reply reference/libxml/setup.xml:49]: Yeah the manual covers the last 2 major versions, and I know it is a PITA :sweat:

**Review by Girgias — APPROVED** [review]

**Comment by Girgias** [comment]
Fixes GH-1362
Fixes GH-2574

**Pull request merged**